packages/core/parcel-bundler/src/builtins/hmr-runtime.js

```diff
@@ -36,11 +36,16 @@ if ((!parent || !parent.isParcelRequire) && typeof WebSocket !== 'undefined') {
         hmrApply(global.parcelRequire, asset);
       });
 
+      var handled = false;
       data.assets.forEach(function (asset) {
         if (!asset.isNew) {
-          hmrAccept(global.parcelRequire, asset.id);
+          handled = handled || hmrAccept(global.parcelRequire, asset.id);
```

Comment:
I think this will cause updates to stop after the first asset accepts. Then handled will be true and hmrAccept won’t get called on other assets. I think we need to ensure that hmrAccept is still called for all assets, and just keep track of handled separately.

Reply:
Fixed,
another way to fix this would have been:
```js
handled = hmrAccept(global.parcelRequire, asset.id) || handled;
```
But I think it doesn't really convey the intention as well (at first glance).